frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -492,6 +507,74 @@ function ReaderFooter:addToMainMenu(menu_items)
     if Device:isAndroid() then
         table.insert(sub_items, getMinibarOption("wifi_status"))
     end
+    table.insert(sub_items, {
+        text = _("Alignment"),
+        sub_item_table = {
+            {
+                text = _("Center"),
+                checked_func = function()
+                    return self.settings.align == "center" or self.settings.align == nil
+                end,
+                callback = function()
+                    self.settings.align = "center"
+                    self.footer_container = CenterContainer:new{
+                        dimen = Geom:new{ w = 0, h = self.height },
+                        self.footer_content,
+                    }
+                    self.footer_positioner = BottomContainer:new{
+                        dimen = Geom:new{},
+                        self.footer_container,
+                    }
+                    self[1] = self.footer_positioner
```

Comment:
That code is duplicated with the one in `ReaderFooter:init()` that you updated above.
Might be moved in a new `ReaderFooter:updateFooterContainer()` so there's a single copy of that code that could be called from both places.
That would give here:
```lua
                callback = function()
                    self.settings.align = "center"
                    self:updateFooterContainer(true)
                    self:resetLayout(true)
                    self:updateFooter()
                    UIManager:setDirty(nil, "ui")
                end,
```

Reply:
I second that.

Reply:
Good point.